sdks/python/apache_beam/ml/inference/vertex_ai_inference.py

```diff
@@ -83,9 +87,15 @@ def __init__(
     """
 
     self._env_vars = kwargs.get('env_vars', {})
+
     # TODO: support the full list of options for aiplatform.init()
     # See https://cloud.google.com/python/docs/reference/aiplatform/latest/google.cloud.aiplatform#google_cloud_aiplatform_init
-    aiplatform.init(project=project, location=location, experiment=experiment)
+    subnetwork = kwargs.get('subnetwork', None)
```

Comment:
I'm a little torn on accepting this as a kwarg or making it an optional named param to the init function

Reply:
Went ahead and made it an optional positional arg instead while making other fixes as well